venafi/resource_venafi_certificate.go

```diff
@@ -356,5 +364,76 @@ func enrollVenafiCertificate(d *schema.ResourceData, cl endpoint.Connector) erro
 
 	d.SetId(req.PickupID)
 	log.Println("Setting up private key")
+
+	KeyPassword := d.Get("key_password").(string)
+	pkcs12_cert, err := AsPKCS12(pcc.Certificate, pcc.PrivateKey, pcc.Chain, KeyPassword)
+
+	if err == nil {
+		s := base64.StdEncoding.EncodeToString(pkcs12_cert)
+		if err = d.Set("pkcs12", s); err != nil {
+			return fmt.Errorf("error setting pkcs12: %s", err)
+		}
+	}
+
 	return d.Set("private_key_pem", pcc.PrivateKey)
 }
+
+func AsPKCS12(certificate string, privateKey string, chain []string, keyPassword string) ([]byte, error) {
+
+	if len(certificate) == 0 || len(privateKey) == 0 {
```

Comment:
empty string in Go is can also be done with testing empty string afaik (but this way - len(string) == 0) is not incorrect)
if certificate == "" || privateKey == ""